routes/recipes-model.js

```diff
@@ -0,0 +1,51 @@
+const db = require('../dbConfig');
+
+module.exports = {
+  find,
+  findById,
+  add,
+  update,
+  remove,
+};
+
+
+function find() {
+  return db('recipes')
+  .join('dishes', 'dishes.id', 'recipes.dish_id' )
+  .select()
+}
+
+function findById(id) {
+  return db('recipes')
+    .where({ id })
+    .first();
+}
+
+function add(dish) {
+  // passing 'id' as the second parameter is recommended to ensure the id is returned
+  // when connecting to other database management systems like Postgres
+  return db('recipes')
+    .insert(dish, 'id')
+    .then(([id]) => {
```

Comment:
You should have another .then to handle the promise from the findById function. That is a database call which is asynchronous.